SourceryRuntime/Sources/AST/TypeName/Tuple_Linux.swift

```diff
@@ -0,0 +1,201 @@
+#if !canImport(ObjectiveC)
+import Foundation
+// For DynamicMemberLookup we need to import Stencil,
+// however, this is different from SourceryRuntime.content.generated.swift, because
+// it cannot reference Stencil
+import Stencil
+
+/// Describes tuple type
+public final class TupleType: NSObject, SourceryModel, Diffable, DynamicMemberLookup {
+    public subscript(dynamicMember member: String) -> Any? {
+        switch member {
+            case "elements":
```

Comment:
what about name? we def need to codegen those things

Reply:
Sure, I have mentioned it in #1198 which would take care of it. I also marked this Linux support release as "experimental" (and I'd like to do the same in the next release notes as well), to make it clear that this is not 100% working as of now. But maintaining large MR is cumbersome, I'd like to give some traction now. 

Apart from that, `DynamicMemberLookup` protocol conformance is isolated to non-Darwin platforms. It is done now to make all tests green.